coralproject/talk
# [CORL-664] pre-mod new commenters

## What it does:
- adds config section to moderation config for "new commenters" (didn't get design link so copy will be revised) allowing admins to enable/disable premoderating new commenters and configure the number of comments that must be approved
- if the setting is enabled, new user's comments will automatically be held for moderation until they have had enough comments approved. 
- new user's comments will be flagged "new commenter" in queue

There are a couple ways I could have done this. I chose not to use the existing premoderate user setting because it could be potentially confusing? I also don't know if it's important to be saving whether the user has passed this threshold rather than counting approved comments each time... open to opinions on this.  

## how to test
- as an admin, enable the feature from moderation settings
- sign up/in to stream as a new user
- make 2 comments, they should be held for moderation
- as admin, approve both comments
- as new user, you should be able to submit a comment without triggering moderation.
